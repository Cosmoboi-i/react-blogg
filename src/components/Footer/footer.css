.footer {
  height: 6%;
  background: black;
  color: white;
  display: flex;
  justify-content: center;
}

.footer-text {
  align-self: center;
  font-size: 0.9em;
}