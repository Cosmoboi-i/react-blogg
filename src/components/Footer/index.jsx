import React from "react";
import "./footer.css";

export default function Footer() {
  return (
    <div className="footer">
      <div className="content">
        <div className="footer-text">@artifact.com 2019</div>
      </div>
    </div>
  );
}
