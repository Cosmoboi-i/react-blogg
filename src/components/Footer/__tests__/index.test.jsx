import { render } from "@testing-library/react";
import React from "react";
import Footer from "..";

describe("Footer", () => {
  it("should render Footer component", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
