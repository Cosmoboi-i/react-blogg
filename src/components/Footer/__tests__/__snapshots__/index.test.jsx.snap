// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer component 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <div
      class="content"
    >
      <div
        class="footer-text"
      >
        @artifact.com 2019
      </div>
    </div>
  </div>
</DocumentFragment>
`;
